File: University/ChopMorabaS1.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {

            int side = scanner.nextInt();

            for(int i = 1; i <= side; i++) {
                if (i == 1) {
                    print(side);
                    System.out.print("");
                }else if (i == side) {
                    print(side);
                }else {
                    printDeep(side);
                }
                System.out.println("");
            }
        }
    }

    public static void print(int side) {
        for (int i = 1; i <= side; i++) {
            System.out.print("*");
        }
    }

    public static void printDeep(int side) {
        for (int i = 1; i <= side; i++) {
            if (i == 1 || i == side) {
                System.out.print("*");
            }else {
                System.out.print(" ");
            }
        }
    }
}

// </Ctrl+Z>

File: University/ChopMorabaS1.py
def main():
    side = int(input())
    for i in range(1, side + 1):
        if i == 1:
            print("*" * side, end="")
        elif i == side:
            print("*" * side)
        else:
            print_deep(side)
        print("")

def print_deep(side):
    print("*" + " " * (side - 2) + "*")

if __name__ == "__main__":
    main()
